packages/atomic/cypress/fixtures/test-fixture.ts

```diff
@@ -243,24 +243,26 @@ export class TestFixture {
         return;
       }
 
-      searchInterfaceComponent.initialize(sampleConfig).then(() => {
```

Comment:
Semi related to the failures in multi-value-text: 

When there's retry, we can end up trying to initialize the search-interface multiple times (which Atomic will prevent, by throwing warnings/errors in the console).

We can prevent that by just checking the `hydrated` class (ie: what stencil generate when a component is rendered)